### PR TITLE
Create twoPowerCurves.py in Python/JAX

### PR DESCRIPTION
**Pull request by Garvit9000c**
index

### CODE REVIEW

**Comment by mjsML** [comment]
@Garvit9000c thanks, please add the other github changes as per my guidelines [here](https://github.com/probml/pyprobml/issues/99#issuecomment-803311745)

**Comment by mjsML** [comment]
Also here is a tip I learned about recently, you can use [proper labels](https://matplotlib.org/stable/tutorials/text/mathtext.html) like 
```python
plt.text(-0.34, -1.8, '$θ_0$', fontsize=12) 
```
[instead of this](https://github.com/probml/pyprobml/pull/301/files#diff-ee1fdf36ec4d08c45481885260de080f3f7219aa5786778ab68c55188d3692abR33)

**Review by mjsML — CHANGES_REQUESTED** [review]
As per my comments.

**Comment by Garvit9000c** [comment]
Ok Sir

**Review by mjsML — APPROVED** [review]
Looks good to me.

**Comment by Garvit9000c** [comment]
Thanks a Lot Sir! (Now onward I will use proper Labeling for Subscripts)

**Review by murphyk — CHANGES_REQUESTED** [review]
Please rewrite in vanilla numpy without using jax.

**Comment by Garvit9000c** [comment]
OK Sir
